### PR TITLE
docs(hx-radio): implement and document vars and attrs

### PR DESCRIPTION
**Pull request by lalithkarikelli**
## Description

`<hx-radio>` : Implement and document attribute and variables

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF 2123

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

**Review by 100stacks — CHANGES_REQUESTED** [review]
### @lalithkarikelli, please verify these updates work in IE11 and Legacy Edge.

**Comment by lalithkarikelli** [comment]
> ### @lalithkarikelli, please verify these updates work in IE11 and Legacy Edge.

@100stacks verified and updated code to work in all 5 browsers . please review

File: src/scss/components/radio/mixins/_hxRadioControl.scss [hunk 1]
```diff
@@ -28,16 +31,17 @@
...
   @if $pseudo-state == focus {
     box-shadow: $focus-glow-invalid;
-  }
-  @else if $pseudo-state == hover {
+  } @else if $pseudo-state == hover {
     &::-ms-check {
       background-color: $red-status-100;
+      background-color: var(--hxRadio-invalid-radio-hover-ie-bgcolor, $red-status-100);
       border: 1px solid currentColor;
       color: $red-status-900;
```
Comment (100stacks) [inline src/scss/components/radio/mixins/_hxRadioControl.scss:39]: Please update to **$red-status-500**.

File: src/scss/components/radio/mixins/_hxRadioControl.scss [hunk 2]
```diff
@@ -28,16 +31,17 @@
...
+      color: var(--hxRadio-invalid-radio-hover-ie-color, $red-status-900);
```
Comment (100stacks) [inline src/scss/components/radio/mixins/_hxRadioControl.scss:40]: Please update to **$red-status-500**.

File: src/scss/components/radio/mixins/_hxRadioControl.scss [hunk 3]
```diff
@@ -28,16 +31,17 @@
...
     }
-  }
-  @else {
+  } @else {
     color: $red-status-900;
```
Comment (100stacks) [inline src/scss/components/radio/mixins/_hxRadioControl.scss:43]: Please update to **$red-status-500**.

File: src/scss/components/radio/mixins/_hxRadioControl.scss [hunk 4]
```diff
@@ -28,16 +31,17 @@
...
+    color: var(--hxRadio-invalid-radio-color, $red-status-900);
```
Comment (100stacks) [inline src/scss/components/radio/mixins/_hxRadioControl.scss:44]: Please update to **$red-status-500**.

File: src/scss/components/radio/mixins/_hxRadioFacade.scss [hunk 5]
```diff
@@ -19,25 +21,31 @@
   }
   @else if $pseudo-state == hover {
     background-color: $red-status-100;
+    background-color: var(--hxRadio-invalid-radio-facade-hover-bgcolor, $red-status-100);
     border: 1px solid currentColor;
     color: $red-status-900;
```
Comment (100stacks) [inline src/scss/components/radio/mixins/_hxRadioFacade.scss:26]: Please update to **$red-status-500**.

File: src/scss/components/radio/mixins/_hxRadioFacade.scss [hunk 6]
```diff
@@ -19,25 +21,31 @@
...
+    color: var(--hxRadio-invalid-radio-facade-hover-color, $red-status-900);
```
Comment (100stacks) [inline src/scss/components/radio/mixins/_hxRadioFacade.scss:27]: Please update to **$red-status-500**.

File: src/scss/components/radio/mixins/_hxRadioFacade.scss [hunk 7]
```diff
@@ -19,25 +21,31 @@
...
   }
   @else {
     border: 1px solid currentColor;
     color: $red-status-900;
```
Comment (100stacks) [inline src/scss/components/radio/mixins/_hxRadioFacade.scss:31]: Please update to **$red-status-500**.

File: src/scss/components/radio/mixins/_hxRadioFacade.scss [hunk 8]
```diff
@@ -19,25 +21,31 @@
...
+    color: var(--hxRadio-invalid-radio-facade-color, $red-status-900);
```
Comment (100stacks) [inline src/scss/components/radio/mixins/_hxRadioFacade.scss:32]: Please update to **$red-status-500**.

**Review by 100stacks — CHANGES_REQUESTED** [review]
### Based on feedback in PR #814, please update invalid state color to **$red-status-500**.

File: src/scss/components/radio/mixins/_hxRadioControl.scss (hunk 1 above)
Comment (lalithkarikelli) [reply src/scss/components/radio/mixins/_hxRadioControl.scss]: Changes have been done, please review.

File: src/scss/components/radio/mixins/_hxRadioControl.scss (hunk 2 above)
Comment (lalithkarikelli) [reply src/scss/components/radio/mixins/_hxRadioControl.scss]: Changes have been done, please review.

File: src/scss/components/radio/mixins/_hxRadioControl.scss (hunk 3 above)
Comment (lalithkarikelli) [reply src/scss/components/radio/mixins/_hxRadioControl.scss]: Changes have been done, please review.

File: src/scss/components/radio/mixins/_hxRadioControl.scss (hunk 4 above)
Comment (lalithkarikelli) [reply src/scss/components/radio/mixins/_hxRadioControl.scss]: Changes have been done, please review.

File: src/scss/components/radio/mixins/_hxRadioFacade.scss (hunk 5 above)
Comment (lalithkarikelli) [reply src/scss/components/radio/mixins/_hxRadioFacade.scss]: Changes have been done, please review.

File: src/scss/components/radio/mixins/_hxRadioFacade.scss (hunk 6 above)
Comment (lalithkarikelli) [reply src/scss/components/radio/mixins/_hxRadioFacade.scss]: Changes have been done, please review.

File: src/scss/components/radio/mixins/_hxRadioFacade.scss (hunk 7 above)
Comment (lalithkarikelli) [reply src/scss/components/radio/mixins/_hxRadioFacade.scss]: Changes have been done, please review.

File: src/scss/components/radio/mixins/_hxRadioFacade.scss (hunk 8 above)
Comment (lalithkarikelli) [reply src/scss/components/radio/mixins/_hxRadioFacade.scss]: Changes have been done, please review.

**Review by BY00565233 — APPROVED** [review]
DEV LGTM

**Review by 100stacks — APPROVED** [review]
DEV LGTM

**Pull request merged**